### PR TITLE
Small cleanup common code of webcmdlets

### PR DESCRIPTION
**Pull request by CarloToso**
# PR Summary
Small cleanup of WebCmdlets

- Where possible use =>
- Remove useless ()
- Add {} to if
- Capital letters on comments
- Merge errors using catch () when ()
- Replace != with is not
- Use interpolated strings

## PR Checklist

- [x] [PR has a meaningful title](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
    - Use the present tense and imperative mood when describing your changes
- [x] [Summarized changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] [Make sure all `.h`, `.cpp`, `.cs`, `.ps1` and `.psm1` files have the correct copyright header](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] This PR is ready to merge and is not [Work in Progress](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---work-in-progress).
    - If the PR is work in progress, please add the prefix `WIP:` or `[ WIP ]` to the beginning of the title (the `WIP` bot will keep its status check at `Pending` while the prefix is present) and remove the prefix when the PR is ready.
- **[Breaking changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#making-breaking-changes)**
    - [x] None
    - **OR**
    - [ ] [Experimental feature(s) needed](https://github.com/MicrosoftDocs/PowerShell-Docs/blob/main/reference/7.3/Microsoft.PowerShell.Core/About/about_Experimental_Features.md)
        - [ ] Experimental feature name(s): 
- **User-facing changes**
    - [x] Not Applicable
    - **OR**
    - [ ] [Documentation needed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
        - [ ] Issue filed: 
- **Testing - New and feature**
    - [x] N/A or can only be tested interactively
    - **OR**
    - [ ] [Make sure you've added a new test if existing tests do not effectively test the code changed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#before-submitting)
- **Tooling**
    - [ ] I have considered the user experience from a tooling perspective and don't believe tooling will be impacted.
    - **OR**
    - [ ] I have considered the user experience from a tooling perspective and opened an issue in the relevant tool repository. This may include:
        - [ ] Impact on [PowerShell Editor Services](https://github.com/PowerShell/PowerShellEditorServices) which is used in the [PowerShell extension](https://github.com/PowerShell/vscode-powershell) for VSCode
        (which runs in a different PS Host).
            - [ ] Issue filed: 
        - [ ] Impact on Completions (both in the console and in editors) - one of PowerShell's most powerful features.
            - [ ] Issue filed: 
        - [ ] Impact on [PSScriptAnalyzer](https://github.com/PowerShell/PSScriptAnalyzer) (which provides linting & formatting in the editor extensions).
            - [ ] Issue filed: 
        - [ ] Impact on [EditorSyntax](https://github.com/PowerShell/EditorSyntax) (which provides syntax highlighting with in VSCode, GitHub, and many other editors).
            - [ ] Issue filed:

### CODE REVIEW

**Comment by xtqqczze** [comment]
Please split the `HttpKnownHeaderNames` changes to a new PR.

File: src/Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/PSUserAgent.cs [hunk 1]
```diff
@@ -24,7 +24,7 @@ internal static string UserAgent
                 string userAgent = string.Format(CultureInfo.InvariantCulture,
                     "{0} ({1}; {2}; {3}) {4}",
                     Compatibility, PlatformName, OS, Culture, App);
-                return (userAgent);
+                return userAgent;
```
Comment (iSazonov) [inline src/Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/PSUserAgent.cs:27]: Please use interpolated strings everywhere is possible.

File: src/Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/WebCmdletElementCollection.cs [hunk 2]
```diff
@@ -22,33 +22,21 @@ internal WebCmdletElementCollection(IList<PSObject> list)
...
         /// <summary>
         /// Finds the element by id.
         /// </summary>
         /// <param name="id"></param>
         /// <returns>Found element as PSObject.</returns>
-        public PSObject FindById(string id)
-        {
-            return Find(id, true);
-        }
+        public PSObject FindById(string id) => Find(id, true);
```
Comment (iSazonov) [inline src/Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/WebCmdletElementCollection.cs:32]: Please use named argument for `true`.

File: src/Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/WebCmdletElementCollection.cs [hunk 3]
```diff
@@ -22,33 +22,21 @@ internal WebCmdletElementCollection(IList<PSObject> list)
...
         /// <summary>
         /// Finds the element by name.
         /// </summary>
         /// <param name="name"></param>
         /// <returns>Found element as PSObject.</returns>
-        public PSObject FindByName(string name)
-        {
-            return Find(name, false);
-        }
+        public PSObject FindByName(string name) => Find(name, false);
```
Comment (iSazonov) [inline src/Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/WebCmdletElementCollection.cs:39]: The same.

**Review by iSazonov — APPROVED** [review]

**Comment by iSazonov** [comment]
@CarloToso  For follow PRs if you have an interest. We could remove `override` parameters in web cmdlets since they duplicate `virtual` parameters in common.cs. Also it would be great to add nullable annotations (string with low level classes).

**Pull request merged**